alex/applications/PublicTransportInfoCS/platform_info.py

```diff
@@ -0,0 +1,126 @@
+# encoding: utf8
+import re
+
+
+class PlatformInfo(object):
+    def __init__(self, from_stop, to_stop, from_city, to_city, train_name):
+        self.from_stop = from_stop
+        self.to_stop = to_stop
+        self.from_city = from_city
+        self.to_city = to_city
+        self.train_name = train_name
+
+    def __unicode__(self):
+        return u"%s, %s -- %s, %s" % (self.from_stop, self.from_city,
+                                     self.to_stop, self.to_city, )
+
+
+class PlatformFinderResult(object):
+    def __init__(self, platform, track, direction):
+        self.platform = platform
+        self.track = track
+        self.direction = direction
+
+    def __unicode__(self):
+        return u"PlatformFinderResult(platform=%s, track=%s, direction=%s)" % (
+            unicode(self.platform),
+            unicode(self.track),
+            unicode(self.direction), )
+
+
+class CRWSPlatformInfo(object):
+    station_name_splitter = re.compile(r'[^a-zěščřžýáíéďťňóůú]*', re.UNICODE)
```

Comment:
If I understand correctly the paragraph for `\W` at https://docs.python.org/2/library/re.html, it won't be locale-dependent unless you set `re.LOCALE` .